app/src/main/res/values-de/strings.xml

```diff
@@ -100,11 +100,13 @@
     <!-- EXPORT -->
     <string name="export_are_you_sure">Bist du sucher, dass du %1$d Dateien, zurück in deine Galerie, <b>exportieren</b> willst?</string>
     <string name="export_are_you_sure_this">Bist du sicher, dass du diese Datei, zurück in deine Galerie, <b>exportieren</b> willst?</string>
+    <string name="export_and_delete_are_you_sure_this">Sind Sie sicher, dass Sie diese Datei in Ihre Galerie <b>exportieren</b> und löschen möchten?</string>
```

Comment:
In german we use "du" and here are some grammar issues

Reply:
Also, please don't use google translate for these translations. Add the english with a <!-- TODO --> in order for the badges in the README to work.